core/src/operations/decorators/advice.rs

```diff
@@ -204,6 +207,7 @@ impl fmt::Display for AdviceInjector {
             Self::SmtGet => write!(f, "smt_get"),
             Self::MemToMap => write!(f, "mem_to_map"),
             Self::HdwordToMap { domain } => write!(f, "hdword_to_map.{domain}"),
+            Self::PSign { sign_kind } => write!(f, "push_sign.{sign_kind}"),
```

Comment:
To be consistent with the previous comment, I'd change this to "sig_to_stack.{kind}".